UnitySDK/Assets/ML-Agents/Scripts/InferenceBrain/Utils/RandomNormal.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+
+namespace MLAgents.InferenceBrain.Utils
```

Comment:
There are also unit tests for the utils classes, might worth porting them over as well.